RetailCoder.VBE/UI/Command/Refactorings/FormDesignerRefactorRenameCommand.cs

```diff
@@ -1,32 +1,30 @@
 using System.Linq;
 using System.Runtime.InteropServices;
-using System.Windows.Forms;
 using Rubberduck.Parsing.Symbols;
 using Rubberduck.Parsing.VBA;
 using Rubberduck.Refactorings.Rename;
 using Rubberduck.UI.Refactorings.Rename;
+using Rubberduck.VBEditor;
 using Rubberduck.VBEditor.SafeComWrappers.Abstract;
 
 namespace Rubberduck.UI.Command.Refactorings
 {
     [ComVisible(false)]
     public class FormDesignerRefactorRenameCommand : RefactorCommandBase
     {
-        private readonly IVBE _vbe;
         private readonly RubberduckParserState _state;
         private readonly IMessageBox _messageBox;
 
         public FormDesignerRefactorRenameCommand(IVBE vbe, RubberduckParserState state, IMessageBox messageBox) 
             : base (vbe)
         {
-            _vbe = vbe;
             _state = state;
             _messageBox = messageBox;
         }
 
         protected override bool EvaluateCanExecute(object parameter)
         {
-            return _state.Status == ParserState.Ready;
+            return _state.Status == ParserState.Ready && GetTarget() != null;
```

Comment:
why can we assume here that `_state` isn't null, but can't assume that in FormDesignerRefactorRenameCommandMenuItem?

Reply:
sorry, missed that one. will nullcheck.

Reply:
The other way around: the RPS will never be null; if it is, things would catch fire well before this code runs ;-)